drivers/pwm/Kconfig.gecko

```diff
@@ -0,0 +1,13 @@
+# GECKO PWM configuration options
+
+# Copyright (c) 2021 Sun Amar.
+# SPDX-License-Identifier: Apache-2.0
+
+config PWM_GECKO
```

Comment:
To automatically enable this driver, if `CONFIG_PWM=y` is set, this block should be appended to the file soc/arm/silabs_exx32/efr32fg1p/Kconfig.defconfig.efr32fg1p :
```
config PWM_GECKO
       default y
       depends on PWM
```